app/controllers/declaration/ConsigneeDetailsController.scala

```diff
@@ -69,7 +69,7 @@ class ConsigneeDetailsController @Inject()(
       case DeclarationType.CLEARANCE =>
         controllers.declaration.routes.DeclarationHolderController.displayPage
       case _ =>
```

Comment:
Should this not always continue to Add. Actors Summary controller?  Which will then redirect to Add page if necessary.

Reply:
Well spotted!